src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/CodeSnippets.cs

```diff
@@ -374,6 +374,54 @@ partial class Test
             }
             """;
 
+        public static string LibraryImportInRefStruct = """
+            using System;
+            using System.Runtime.InteropServices;
+
+            public static partial class MyClass
+            {
+                public ref partial struct RSPublic
+                {
+                    [LibraryImport("DNE")]
```

Comment:
I'm usually okay with acrunyms like `"NDE"`, but it does tend to breed the question of what that means? Since this is in tests and the point is for them to be read, I think it generally appropriate to avoid the non-standard acronym. Since there is no context about that string I would prefer `"DoesNotExist"`.